quacc/calculators/qchem.py

```diff
@@ -0,0 +1,100 @@
+"""
+A wrapper around ASE's Q-Chem calculator that makes it better suited for high-throughput DFT.
+"""
+from __future__ import annotations
+
+import inspect
+import os
+import warnings
+
+import numpy as np
+from ase import Atoms
+import ase.units
+from ase.calculators.calculator import FileIOCalculator
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.io.qchem.sets import ForceSet
+
+from quacc import SETTINGS
+from quacc.custodian import qchem as custodian_qchem
+
+
+class QChem(FileIOCalculator):
```

Comment:
The convention in Quacc is for all classes to be defined using a `@dataclass` wrapper so you don't have self assign all the variables. I will make that change for you later.